batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/entity/Admin.java

```diff
@@ -1,5 +1,7 @@
 package com.lftechnology.batch7crud.entity;
 
+import java.util.Date;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Date'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)